packages/ui/src/components/va-image/VaImage.vue

```diff
@@ -49,7 +49,7 @@ export default defineComponent({
     draggable: { type: Boolean, default: true },
   },
   setup (props, { emit }) {
-    const loading = ref(true)
+    const loading = ref(false)
```

Comment:
Since issues related to SSR only we can stick to isServer

```suggestion
    const loading = ref(isClient())
```

Reply:
We either get warns on hydration or image not being shown.